src/bin/lockin.rs

```diff
@@ -257,7 +268,13 @@ const APP: () = {
                         }
                         Conf::InPhase => output.re >> 16,
                         Conf::Quadrature => output.im >> 16,
-                        Conf::Modulation => DAC_SEQUENCE[i] as i32,
+
+                        // Note: Because the signal generator has a period equal to one batch size,
+                        // it's okay to only update it when outputting the modulation waveform, as
```

Comment:
Yeah, the implication of that is that the generated signal "pauses" when it is not being output instead of logically continuing as an output that is not selected. E.g. if you had a very low frequency sinusoid, and you swapped outputs for a while, when you swapped back, it would resume where it previously left off. I don't think that's a problem though.